packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -168,6 +168,8 @@ abstract class Logger {
   /// Only surfaces a value in machine modes, Loggers may ignore this message in
   /// non-machine modes.
   void sendEvent(String name, [Map<String, dynamic> args]) { }
+
```

Comment:
As with the other pubic methods of this interface, please add doc comments.

Reply:
Added one in the latest commit, and also removed the comment from `BufferLogger.clear`.